website/content/docs/api-gateway/upgrade-specific-versions.mdx

```diff
@@ -9,11 +9,186 @@ description: >-
 
 This topic describes how to upgrade Consul API Gateway.
 
-## Breaking Changes
+
+## v0.3.0
+
+Consul API Gateway v0.3.0 introduces a breaking change for people upgrading from lower versions. Gateways with a `secret` defined in a different namespace now require a [`ReferencePolicy`](https://gateway-api.sigs.k8s.io/v1alpha2/references/spec/#gateway.networking.k8s.io/v1alpha2.ReferencePolicy) that explicitly allows `Gateways` from the gateway's namesapce to use `Secrets` in the secret's namespace.
```

Comment:
Let's avoid "introduces a breaking change", @trujillo-adam can you help with the messaging?